bigquery_magics/bigquery.py

```diff
@@ -97,7 +97,7 @@
 from google.api_core import client_info
 from google.api_core.exceptions import NotFound
 from google.cloud import bigquery
-from google.cloud.bigquery import exceptions
+from google.cloud.bigquery import _versions_helpers, exceptions
```

Comment:
Please don't.

That's a private module. Let's use the one we copied to this repo, instead.